content/en/docs/concepts/storage/volumes.md

```diff
@@ -1247,6 +1247,13 @@ in `containers[*].volumeMounts`. Its values are:
   (unmounted) by the containers on termination.
   {{< /warning >}}
 
+  {{< warning >}}
```

Comment:
I'd use a caution here, not a warning. But also, I'd move this to the start of the section, just under the heading.

Reply:
done